packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -360,12 +360,28 @@ export class HttpPlugin extends BasePlugin {
 
       const setter: HeaderSetter = {
         setHeader(name: string, value: string) {
-          request.setHeader(name, value);
+          // https://github.com/googleapis/cloud-trace-nodejs/pull/766
```

Comment:
What would you think about removing this PR reference and instead putting a short explanation of the logic?